docs/Storage Rules.md

```diff
@@ -0,0 +1,190 @@
+# Storage Rules
+
+Storage rules are based on Firebase. You can see the [pull request here](https://github.com/nhost/hasura-backend-plus/pull/144).
+
+The rules are governed by a `yaml` file, which defines routes and permissions, with authenticating functions. You can see [the template file here](https://github.com/nhost/hasura-backend-plus/blob/master/custom/storage-rules/rules.yaml)
```

Comment:
"governed" sounds complex. How about "The rules are placed in a `yaml` file"? (just an example)

"with authenticating functions."

I think the correct technical term is "with authorization functions".

https://stackoverflow.com/a/6556548/885968